.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Just curious as to why you are using eslint instead of tslint?

Reply:
my understanding is that TSLint is going away and will be replaced with eslint:

https://medium.com/palantir/tslint-in-2019-1a144c2317a9